core-backend/common/src/funcs.rs

```diff
@@ -49,7 +49,7 @@ pub fn get_u128(mem: &dyn Memory, ptr: usize) -> Result<u128, &'static str> {
 pub fn get_vec(mem: &dyn Memory, ptr: usize, len: usize) -> Result<Vec<u8>, &'static str> {
     let mut vec = vec![0u8; len];
     mem.read(ptr, &mut vec)
-        .map_err(|_| "Cannot read bytes from mem")?;
+        .map_err(|_| "Cannot read vector from mem")?;
```

Comment:
I think we still read bytes

Reply:
done [762e0be](https://github.com/gear-tech/gear/pull/853/commits/762e0be06f9f42026e2db7bff5a957ed5c71e9d0)